jkube-kit/build/service/docker/src/test/java/org/eclipse/jkube/kit/build/service/docker/helper/ConfigHelperTest.java

```diff
@@ -30,16 +31,27 @@
 import java.util.Properties;
 
 import static org.assertj.core.api.AssertionsForInterfaceTypes.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.RETURNS_DEEP_STUBS;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
 
 public class ConfigHelperTest {
-  @Mocked
   private ImageConfigResolver imageConfigResolver;
-
-  @Mocked
   private KitLogger logger;
-
-  @Mocked
   private JavaProject javaProject;
+  private JKubeConfiguration jKubeConfiguration;
+
+  @Before
+  public void setUp() {
+    imageConfigResolver = mock(ImageConfigResolver.class, RETURNS_DEEP_STUBS);
+    logger = mock(KitLogger.SilentLogger.class, RETURNS_DEEP_STUBS);
```

Comment:
https://github.com/eclipse/jkube/pull/1192#discussion_r784889688

The idea is not to mock the logger unless we need to verify something.
```suggestion
    logger = new KitLogger.SilentLogger();
```